src/platform/graphics/graphics-device.js

```diff
@@ -809,6 +817,47 @@ class GraphicsDevice extends EventHandler {
      */
     frameEnd() {
     }
+
+    /**
+     * Get a renderable HDR pixel format supported by the graphics device.
+     *
+     * @param {number[]} [formats] - An array of pixel formats to check for support. Can contain:
+     *
+     * - {@link PIXELFORMAT_111110F}
+     * - {@link PIXELFORMAT_RGBA16F}
+     * - {@link PIXELFORMAT_RGBA32F}
+     *
+     * @param {boolean} [filterable] - If true, the format aso needs to be filterable. Defaults to
+     * true.
+     * @returns {number|undefined} The first supported renderable HDR format or undefined if none is
+     * supported.
+     */
+    getRenderableHdrFormat(formats = [PIXELFORMAT_111110F, PIXELFORMAT_RGBA16F, PIXELFORMAT_RGBA32F], filterable = true) {
```

Comment:
Just wondering whether for API simplicity whether it's better to just have:

```
const supported = device.isRenderable(PIXELFORMAT_111110F);
```

Reply:
These are optional parameters, and typically the user would not pass any, to get any format from smallest size to largest.

It's similar to the device creation where we specify GL1, GL2 and GPU array in any other. Simple to call without having to implement logic on the user side.

Reply:
I already call this function twice in this code, and one more time in follow up PR .. and it's convenient to not have to implement format logic at each place.